website/static/js/fangorn.js

```diff
@@ -261,6 +324,11 @@ function _fangornSending(treebeard, file, xhr, formData) {
     xhr.send = function() {
         _send.call(xhr, file);
     };
+    var filesArr = treebeard.dropzone.getQueuedFiles();
+    if (filesArr.length  > 1) {
```

Comment:
That's a good point, currently not but it's not very hard to do. But this might require a revision of the whole approach. I'll think about it, good point!